### PR TITLE
Feat/channels

### PR DESCRIPTION
**Pull request by huytgps**

### CODE REVIEW

File: src/pages/channels.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// anything
```
Comment (huyhavmodev) [inline src/pages/channels.js:1]: why?

**Review by huyhavmodev — APPROVED** [review]

File: src/pages/channels.js (hunk 1 above)
Comment (huytgps) [reply src/pages/channels.js:1]: reason

**Review by huyhavmodev — CHANGES_REQUESTED** [review]
Request changes

**Comment by huytgps** [comment]
nothing

**Review by huyhavmodev — APPROVED** [review]

**Pull request merged**